tests/_support/rest-api-tester/assets/js/src/trap-script.ts

```diff
@@ -0,0 +1,180 @@
+(
```

Comment:
Ah, the JS above was generated due to TypeScript compilation, perhaps? Side note: we haven't standardized on TypeScript amongst the team so I hesitate at its usage here. That being said, I'm pretty open to a discussion around it. We've talked about adding ES6/Babel to our world - perhaps we do TypeScript instead? I know @barryhughes has been pumped about TypeScript for a while.